src/ecs/src/world.rs

```diff
@@ -27,23 +28,43 @@ impl World {
     /// Creates a new entity in the world and returns a handle to it.
     pub fn create_entity(&mut self) -> Entity {
         let id = self.entities.create();
-        self.components.insert(id.clone(), AnyVec::new());
         id
     }
 
     /// Destroys a given entity and deallocates its components.
```

Comment:
With the removal of line 36, this comment is now incorrect, since no component deallocation is actually done. Either change the comment, or add: `TODO: Add component removal capabilities to DynVec.`